lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,191 @@
 // lms - views - shopping cart
 // ====================
 
+$cart-list-border: 1px solid $border-color-1;
+$hr-border: 1px solid $dark-gray2;
+$input-border: 2px solid $dark-gray2;
+$discount-border: 2px solid $light-gray1;
+$steps-border: 3px solid $light-gray1;
+$light-border: 1px solid $gray-l5;
+$submit-hover-color: #1F8FC2;
+$course-title-color: #4A4A46;
+$shopping-cart-input-color: #666;
+$enrollment-text-color: #9B9B93;
+$contact-support-background-color: #9B9B9B;
+$empty-cart-text-color: #9d9d9d;
+$expiry-border-color: #f0f0f0;
+$card-errors-background: #FFEEF5;
+
+.pull-right {
+  @include float(right);
+}
+
+.fa-caret-right {
+  // flip the icon for RTL
+  &:before {
+    @include rtl {
+      content: "\f0d9"; // FA icon arrow to the right
+    }
+  }
+}
+
 .notification {
   padding: ($baseline*1.5) ($baseline*1.5) 0 ($baseline*1.5);
 }
 
 .error_msg {
   margin: $baseline;
+  border: 1px solid $red;
   padding: ($baseline/4);
   color: $red;
-  border: 1px solid $red;
-
 }
 
-.cart-errors{
-  background: #FFEEF5;
-  color:#B72667;
-  text-align: center;
-  padding: ($baseline/2) 0;
-  font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-  font-size: 15px;
-  border-bottom: 1px solid #B72667;
-  margin-bottom: $baseline;
+.cart-errors {
+  @include font-size(15);
   display: none;
-}
-
-.cart-list {
```

Comment:
I couldn't find .cart-list anywhere in templates, so I removed it since it seemed like a remnants of old style. I need a confirmation that this isn't used somewhere outside of the edx-platform core templates (like on edx.org, for instance).